`npm install` fails

Running `npm install` in a fresh clone of the Eve repository fails for me with the following message:

```
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts will run only for `npm publish`.
npm WARN prepublish-on-install (In npm@4 and previous versions, it also runs for `npm install`.)
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.

> witheve@0.2.3 prepublish /Users/jules/src/Eve
> tsc && node ./build/scripts/build.js

src/renderer.ts(628,52): error TS2339: Property 'value' does not exist on type 'Element'.

npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v7.6.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! witheve@0.2.3 prepublish: `tsc && node ./build/scripts/build.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the witheve@0.2.3 prepublish script 'tsc && node ./build/scripts/build.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the witheve package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     tsc && node ./build/scripts/build.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs witheve
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls witheve
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jules/src/Eve/npm-debug.log
```

I tested this with a fresh clone on macOS Sierra 10.12.3 and on Ubuntu 16.10. Both run the most recent version of node and npm that I could reasonably get (7.6.0 and 4.1.2 respectively). This also happenes with the 0.2.3 source code for me under the same systems, though.

--- a/src/renderer.ts
+++ b/src/renderer.ts
@@ -625,7 +625,7 @@ window.addEventListener("change", function(event) {
     let value = target.value;
 
     if(isSelectElem(target)) {
-      value = target.options[target.selectedIndex].value;
+      value = target.options.item(target.selectedIndex).value;
     }
 
     sentInputValues[target.entity!].push(value);

